src/lucky/cookies/flash_store.cr

```diff
@@ -22,6 +22,10 @@ class Lucky::FlashStore
     raise Lucky::InvalidFlashJSONError.new(session.get?(SESSION_KEY))
   end
 
+  def keep : Void
```

Comment:
We used `Void` in a few places, but typically that is reserved for doing C extensions and "idiomatic Crystal" is to use `Nil`. I don't think there is a technical difference, but 🤷  

```suggestion
  def keep : Nil
```

Reply:
Ok, good to know! I wasn't sure, but I saw it in some other places so I used it for consistency.